tdm-be/src/swagger/swagger-config.json

```diff
@@ -103,7 +103,7 @@
             "example": [
               {
                 "url": "https://data-computer.services.istex.fr",
-                "retrieveUrl": "/v1/retrieve",
+                "retrieveUrl": {"url": "/v1/retrieve-csv", "fileExtension": "csv"},
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Would `.tar.gz` be a valid extension?

If we wanted to use the original `v1/retrieve` URL...

Reply:
Yes, this is use to complete the file extension of the retrieved file
https://github.com/Inist-CNRS/tdm-factory/blob/a3abd6075655b9f06d10b355eea3b9ec9c09cf9c/tdm-be/src/worker/enrichmentHook.ts#L113